apps/cowswap-frontend/src/modules/account/containers/AccountDetails/index.tsx

```diff
@@ -195,7 +200,8 @@ export function AccountDetails({
 
   const networkLabel = NETWORK_LABELS[chainId]
 
-  const isHardWareWallet = getIsHardWareWallet(connectionType.type)
+  // TODO: Remove when done testing
+  const isHardWareWallet = 1 < 2 ? true : getIsHardWareWallet(connectionType.type)
```

Comment:
Lets not forget :) 

There's probably a way to add a special comment and block PRs from merging when we do these things. So easy to forget otherwise

Reply:
Fixed